### ISSUE TITLE
Grafana 10.2.3 is breaking dashboard (No data)

### ISSUE THREAD

**Issue by jostrasser** [grafana/grafana#79692]
### What happened?

After updating from 10.2.2 to 10.2.3 my Syslog dashboard is broken on specific devices.
No data shown in Time series and Table panels, if I select/filter one/all of the following hostnames: `UniFi-UAP-EG-1, UniFi-UAP-OG-1, UniFi-UAP-G-1 or UniFi-USW-EG-1`.
Downgrading to 10.2.2 is fixing this issue immediately.
If required I can upload the full dashboard.

**10.2.2:**
![SYSLOG MONITORING - HOMENET - Dashboards - Grafana 2](https://github.com/grafana/grafana/assets/14274103/c257f97d-05c8-4427-89ce-5c7cf167c92c)

**10.2.3:**
![SYSLOG MONITORING - HOMENET - Dashboards - Grafana](https://github.com/grafana/grafana/assets/14274103/dc7ebfd2-253e-4ae7-a6c8-92720d8d410f)

### What did you expect to happen?

Working dashboard without impact on filtering on hostnames.

### Did this work before?

Yes, on all version before 10.2.3.

### How do we reproduce it?

1. Update to 10.2.3
2. Filter one of the following hostnames (see screenshot)
3. No data will be displayed

### Is the bug inside a dashboard panel?

| Key | Value |
|--|--|
| Panel | timeseries @ 8.0.3 |
| Grafana | 10.2.3 (1e84fede543acc892d2a2515187e545eb047f237) // Open Source |
<details><summary>Panel debug snapshot dashboard</summary>

```json
{
  "panels": [
    {
      "datasource": {
        "type": "grafana",
        "uid": "grafana"
      },
      "fieldConfig": {
        "defaults": {
          "custom": {
            "drawStyle": "bars",
            "lineInterpolation": "linear",
            "barAlignment": 0,
            "lineWidth": 1,
            "fillOpacity": 100,
            "gradientMode": "none",
            "spanNulls": false,
            "insertNulls": false,
            "showPoints": "never",
            "pointSize": 5,
            "stacking": {
              "mode": "none",
              "group": "A"
            },
            "axisPlacement": "right",
            "axisLabel": "MESSAGES / MIN",
            "axisColorMode": "text",
            "axisBorderShow": false,
            "scaleDistribution": {
              "type": "linear"
            },
            "axisCenteredZero": false,
            "hideFrom": {
              "tooltip": false,
              "viz": false,
              "legend": false
            },
            "thresholdsStyle": {
              "mode": "off"
            },
            "axisGridShow": true
          },
          "color": {
            "mode": "palette-classic"
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green",
                "value": null
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          },
          "decimals": 0,
          "unit": "none"
        },
        "overrides": [
          {
            "matcher": {
              "id": "byName",
              "options": "Info"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "rgb(80, 80, 80)",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Notice"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "rgb(182, 182, 182)",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Warning"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "#E0B400",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Error"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "#FF780A",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Critical"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "#E02F44",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Alert"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "#8F3BB8",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          },
          {
            "matcher": {
              "id": "byName",
              "options": "Emergency"
            },
            "properties": [
              {
                "id": "color",
                "value": {
                  "fixedColor": "#8F3BB8",
                  "mode": "fixed"
                }
              },
              {
                "id": "custom.stacking",
                "value": {
                  "group": "A",
                  "mode": "normal"
                }
              }
            ]
          }
        ]
      },
      "gridPos": {
        "h": 13,
        "w": 15,
        "x": 0,
        "y": 0
      },
      "id": 2,
      "interval": "",
      "links": [],
      "options": {
        "tooltip": {
          "mode": "multi",
          "sort": "none"
        },
        "legend": {
          "showLegend": true,
          "displayMode": "table",
          "placement": "right",
          "calcs": [
            "last",
            "max",
            "sum"
          ]
        }
      },
      "pluginVersion": "8.0.3",
      "targets": [
        {
          "refId": "A",
          "datasource": {
            "type": "grafana",
            "uid": "grafana"
          },
          "queryType": "snapshot",
          "snapshot": []
        }
      ],
      "title": "Reproduced with embedded data",
      "type": "timeseries"
    },
    {
      "gridPos": {
        "h": 7,
        "w": 9,
        "x": 15,
        "y": 0
      },
      "id": 5,
      "options": {
        "content": "<table width=\"100%\">\n    <tr>\n      <th width=\"2%\">Panel</th>\n      <td >timeseries @ 8.0.3</td>\n    </tr>\n    <tr>\n      <th>Queries</th>\n      <td>A[influxdb], B[influxdb], D[influxdb], C[influxdb], E[influxdb], F[influxdb], G[influxdb]</td>\n    </tr>\n    \n    <tr><th>Data</th><td> 0 frames, 0 fields, 0 rows </td></tr>\n    \n    <tr>\n      <th>Grafana</th>\n      <td>10.2.3 (1e84fede543acc892d2a2515187e545eb047f237) // Open Source</td>\n    </tr>\n  </table>",
        "mode": "html"
      },
      "title": "Debug info",
      "type": "text"
    },
    {
      "id": 6,
      "title": "Original Panel JSON",
      "type": "text",
      "gridPos": {
        "h": 13,
        "w": 9,
        "x": 15,
        "y": 7
      },
      "options": {
        "content": "{\n  \"datasource\": {\n    \"type\": \"influxdb\",\n    \"uid\": \"ljjTcu7Mz\"\n  },\n  \"fieldConfig\": {\n    \"defaults\": {\n      \"custom\": {\n        \"drawStyle\": \"bars\",\n        \"lineInterpolation\": \"linear\",\n        \"barAlignment\": 0,\n        \"lineWidth\": 1,\n        \"fillOpacity\": 100,\n        \"gradientMode\": \"none\",\n        \"spanNulls\": false,\n        \"insertNulls\": false,\n        \"showPoints\": \"never\",\n        \"pointSize\": 5,\n        \"stacking\": {\n          \"mode\": \"none\",\n          \"group\": \"A\"\n        },\n        \"axisPlacement\": \"right\",\n        \"axisLabel\": \"MESSAGES / MIN\",\n        \"axisColorMode\": \"text\",\n        \"axisBorderShow\": false,\n        \"scaleDistribution\": {\n          \"type\": \"linear\"\n        },\n        \"axisCenteredZero\": false,\n        \"hideFrom\": {\n          \"tooltip\": false,\n          \"viz\": false,\n          \"legend\": false\n        },\n        \"thresholdsStyle\": {\n          \"mode\": \"off\"\n        },\n        \"axisGridShow\": true\n      },\n      \"color\": {\n        \"mode\": \"palette-classic\"\n      },\n      \"mappings\": [],\n      \"thresholds\": {\n        \"mode\": \"absolute\",\n        \"steps\": [\n          {\n            \"color\": \"green\",\n            \"value\": null\n          },\n          {\n            \"color\": \"red\",\n            \"value\": 80\n          }\n        ]\n      },\n      \"decimals\": 0,\n      \"unit\": \"none\"\n    },\n    \"overrides\": [\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Info\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"rgb(80, 80, 80)\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Notice\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"rgb(182, 182, 182)\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Warning\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"#E0B400\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Error\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"#FF780A\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Critical\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"#E02F44\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Alert\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"#8F3BB8\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      },\n      {\n        \"matcher\": {\n          \"id\": \"byName\",\n          \"options\": \"Emergency\"\n        },\n        \"properties\": [\n          {\n            \"id\": \"color\",\n            \"value\": {\n              \"fixedColor\": \"#8F3BB8\",\n              \"mode\": \"fixed\"\n            }\n          },\n          {\n            \"id\": \"custom.stacking\",\n            \"value\": {\n              \"group\": \"A\",\n              \"mode\": \"normal\"\n            }\n          }\n        ]\n      }\n    ]\n  },\n  \"gridPos\": {\n    \"h\": 7,\n    \"w\": 24,\n    \"x\": 0,\n    \"y\": 1\n  },\n  \"id\": 10,\n  \"interval\": \"\",\n  \"links\": [],\n  \"options\": {\n    \"tooltip\": {\n      \"mode\": \"multi\",\n      \"sort\": \"none\"\n    },\n    \"legend\": {\n      \"showLegend\": true,\n      \"displayMode\": \"table\",\n      \"placement\": \"right\",\n      \"calcs\": [\n        \"last\",\n        \"max\",\n        \"sum\"\n      ]\n    }\n  },\n  \"pluginVersion\": \"8.0.3\",\n  \"targets\": [\n    {\n      \"alias\": \"Info\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"A\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"info\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Notice\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"B\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"notice\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Warning\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"D\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"warning\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Error\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"C\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"err\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Critical\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"E\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"crit\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Alert\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"F\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"alert\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    },\n    {\n      \"alias\": \"Emergency\",\n      \"datasource\": {\n        \"type\": \"influxdb\",\n        \"uid\": \"ljjTcu7Mz\"\n      },\n      \"groupBy\": [\n        {\n          \"params\": [\n            \"1m\"\n          ],\n          \"type\": \"time\"\n        },\n        {\n          \"params\": [\n            \"null\"\n          ],\n          \"type\": \"fill\"\n        }\n      ],\n      \"measurement\": \"syslog\",\n      \"orderByTime\": \"ASC\",\n      \"policy\": \"default\",\n      \"refId\": \"G\",\n      \"resultFormat\": \"time_series\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"severity_code\"\n            ],\n            \"type\": \"field\"\n          },\n          {\n            \"params\": [],\n            \"type\": \"count\"\n          }\n        ]\n      ],\n      \"tags\": [\n        {\n          \"key\": \"severity\",\n          \"operator\": \"=\",\n          \"value\": \"emerg\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"hostname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$hostname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"severity\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$severity$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"appname\",\n          \"operator\": \"=~\",\n          \"value\": \"/^$appname$/\"\n        },\n        {\n          \"condition\": \"AND\",\n          \"key\": \"message\",\n          \"operator\": \"=~\",\n          \"value\": \"/$Query/\"\n        }\n      ]\n    }\n  ],\n  \"title\": \"SYSLOG COUNT\",\n  \"type\": \"timeseries\"\n}",
        "mode": "code",
        "code": {
          "language": "json",
          "showLineNumbers": true,
          "showMiniMap": true
        }
      }
    },
    {
      "id": 3,
      "title": "Data from panel above",
      "type": "table",
      "datasource": {
        "type": "datasource",
        "uid": "-- Dashboard --"
      },
      "gridPos": {
        "h": 7,
        "w": 15,
        "x": 0,
        "y": 13
      },
      "options": {
        "showTypeIcons": true
      },
      "targets": [
        {
          "datasource": {
            "type": "datasource",
            "uid": "-- Dashboard --"
          },
          "panelId": 2,
          "withTransforms": true,
          "refId": "A"
        }
      ]
    }
  ],
  "schemaVersion": 37,
  "title": "Debug: SYSLOG COUNT // 2023-12-19 15:08:12",
  "tags": [
    "debug",
    "debug-timeseries"
  ],
  "time": {
    "from": "2023-12-19T11:08:12.960Z",
    "to": "2023-12-19T14:08:12.960Z"
  }
}
```
</details>

### Environment (with versions)?

Grafana: 10.2.3
OS: Ubuntu Focal
Browser: Safari, Edge

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

### Datasource(s)?

InfluxDB

**Comment by lonionss** [comment]
+1
Raised an identical issue to this with v10.2.2 with Feature Flag 'influxdbBackendMigration' enabled. https://github.com/grafana/grafana/issues/79368
Also tested this and seeing the same behaviour in v10.2.3.

**Comment by jostrasser** [comment]
Thanks @lonionss !
Changing the 'influxdbBackendMigration' FF disabled or enabled did not change the behavior for me. I am always getting a "No data" back. :(

**Comment by vincejv** [comment]
@jostrasser same issue, broke on v10.2.3 along with this issue https://github.com/grafana/grafana/issues/79797
@lonionss seems like your issue is unrelated, i didn't have this issue in v10.2.2

**Comment by lonionss** [comment]
@vincejv Reading through the changelog, looks like the Feature Flag which caused issues in v10.2.2 has been enabled by default in v10.2.3 https://github.com/grafana/grafana/issues/76641.
So it's possible it's related and maybe the reason these issues are resolved when downgrading to v10.2.2?

**Comment by mbentley** [comment]
Thanks for the suggestions for trying `influxdbBackendMigration`.  Setting that to `false` fixed my issue with variables with special characters not being properly escaped and my dashboards are showing data again.

**Comment by juanicabanas** [comment]
Hi! Sorry the delay.
I think this is something @grafana/observability-metrics or @grafana/dashboards-squad can help on

**Comment by gtk-grafana** [comment]
Duplicate of https://github.com/grafana/grafana/issues/80003

**Comment by gtk-grafana** [comment]
This was incorrectly closed, this one is not a duplicate of #80003, going to use this as the canonical issue for the regression in 10.2.2

**Comment by gtk-grafana** [comment]
@jostrasser I'm having some difficulty reproducing this, can you provide the raw query string or screenshot of the query builder?

**Comment by jostrasser** [comment]
> @jostrasser I'm having some difficulty reproducing this, can you provide the raw influx query string or screenshot of the query builder?

Hi, I removed a few panels which are not relevant and exported the dashboard for you:
[SYSLOG MONITORING-1704925110828.json](https://github.com/grafana/grafana/files/13895117/SYSLOG.MONITORING-1704925110828.json)

I hope this can help!

FYI: I tried a few times to get it working with down and upgrading, testing the `influxdbBackendMigration` set to `true` and `false`.
However, now it looks like I fixed it with `influxdbBackendMigration = false`, but I cannot understand why it is working now on version 10.2.3.

It definitely doesn´t work last week and I´ve done the same. :/

**Comment by vincejv** [comment]
@gtk-grafana 
`influxdbBackendMigration = false` or true doesn't change a thing

raw influx query string: 

```sql
SELECT max("result_code") FROM "http_response" WHERE ("server"::tag =~ /^$website_for_conn_check$/) AND $timeFilter GROUP BY time($__interval), "server"::tag fill(none)
```

Query inspector 
```json
{
  "request": {
    "url": "api/ds/query?ds_type=influxdb&requestId=Q104",
    "method": "POST",
    "data": {
      "queries": [
        {
          "datasource": {
            "type": "influxdb",
            "uid": "efe65b41-a338-476b-bbb3-7ad57f64ff86"
          },
          "groupBy": [
            {
              "params": [
                "$__interval"
              ],
              "type": "time"
            },
            {
              "params": [
                "server::tag"
              ],
              "type": "tag"
            },
            {
              "params": [
                "none"
              ],
              "type": "fill"
            }
          ],
          "measurement": "http_response",
          "orderByTime": "ASC",
          "policy": "default",
          "refId": "A",
          "resultFormat": "time_series",
          "select": [
            [
              {
                "params": [
                  "result_code"
                ],
                "type": "field"
              },
              {
                "params": [],
                "type": "max"
              }
            ]
          ],
          "tags": [
            {
              "key": "server::tag",
              "operator": "=~",
              "value": "/^https://disqus.com$/"
            }
          ],
          "adhocFilters": [],
          "query": "",
          "rawSql": "",
          "alias": "",
          "limit": "",
          "slimit": "",
          "tz": "",
          "datasourceId": 4,
          "intervalMs": 60000,
          "maxDataPoints": 1888
        }
      ],
      "from": "1704841893180",
      "to": "1704928293180"
    },
    "hideFromInspector": false
  },
  "response": {
    "results": {
      "A": {
        "error": "InfluxDB returned error: ",
        "errorSource": "",
        "status": 500,
        "refId": "A"
      }
    }
  }
}
```

**Comment by vincejv** [comment]
@gtk-grafana 
Dashboard Json file: [uptime.json](https://github.com/grafana/grafana/files/13895466/uptime.json)
Dashboard, and HTTP Monitoring rows for this dashboard not working at all, **the only fix is to downgrade to 10.2.2**
